package.json

```diff
@@ -18,7 +18,8 @@
     "cookies": "0.7.1",
     "debug": "3.1.0",
     "on-headers": "~1.0.1",
-    "safe-buffer": "5.1.1"
+    "safe-buffer": "5.1.1",
+    "keygrip": "~1.0.2"
```

Comment:
So `npm` will reorder this the next time `npm install` is executed. I suppose you added this by hand rather than with `npm`. Just alphabetize it to keep with the way `npm` will serialize it 👍